#160528936 ft-fetch-single-order

Johnpaul-Borree opened the pull request:
#### What does this PR do?
Create Api endpoint to allow users fetch single order
#### Description of Task to be completed?
- Create a controller action for this feature
- Validate for invalid Id
- Create a route to controller
#### How should this be manually tested?
Test endpoint by starting server and navigating to GET /api/v1/orders/ordersId
#### What are the relevant pivotal tracker stories?
#160528936
#### Screenshots (if appropriate)
![screenshot 17](https://user-images.githubusercontent.com/25976032/45578582-58cab280-b87a-11e8-9815-801a116ec9d5.png)

Johnpaul-Borree committed:
[ft-#160528936]create Endpoint Fetch single order

- create controller action for this action
- validate this endpoint for wrong endpoint
- create route to this controller

KaiserPhemi commented on .eslintrc.json:
```diff
@@ -0,0 +1,29 @@
+            "always"
+        ]
+    }
+}
```
#### Nitpick
New line

Johnpaul-Borree replied:
finished implementing this feedback just now, thanks

KaiserPhemi commented on test/ordersTests.js:
```diff
@@ -0,0 +1,93 @@
+				});
+		});
+	});
+});
```
#### Nitpick
New line

Johnpaul-Borree replied:
implemented, thanks

KaiserPhemi reviewed (changes requested):
Implement feedback

Johnpaul-Borree committed:
[#160528936] implement feedback

- Modify .eslintrc.json for Airbnb style guides
- Refactor code to use Airbnb
- Remove functions
- add newline to end
- add spaces to objects in curly braces

Johnpaul-Borree committed:
Merge branch 'develop' into ft-fetch-single-order-#160528936
